crt_portal/crt_portal/storage.py

```diff
@@ -0,0 +1,16 @@
+from django.core.files.storage import get_storage_class
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class CachedS3Boto3Storage(S3Boto3Storage):
+    """
+    Save compressed files to local storage before uploading to S3.
+    """
+    def __init__(self, *args, **kwargs):
+        super(CachedS3Boto3Storage, self).__init__(*args, **kwargs)
+        self.local_storage = get_storage_class("compressor.storage.CompressorFileStorage")()
+
+    def save(self, name, content):
+        self.local_storage._save(name, content)
+        super(CachedS3Boto3Storage, self).save(name, self.local_storage._open(name))
```

Comment:
Quick fix to update this usage of super
```suggestion
        super().__init__(*args, **kwargs)
        self.local_storage = get_storage_class("compressor.storage.CompressorFileStorage")()

    def save(self, name, content):
        self.local_storage._save(name, content)
        super().save(name, self.local_storage._open(name))
```

Reply:
Thanks for both suggestions! I had `S3ManifestStaticStorage` locally but did not commit it :man_facepalming: Going through the paces once more to make sure everything is kosher.